### ISSUE TITLE
Do you have plan release iOS version ?

### ISSUE THREAD

**Issue by suaron** [xuhcc/beancount-mobile#4]

**Comment by xuhcc** [comment]
Yes, I would like to do that at some point, but it will likely be limited to source code release, because I don't own Apple devices myself and do not plan to pay for Apple Developer Account. However, I wouldn't mind if someone else publishes Beancount Mobile to the app store.

**Comment by johan456789** [comment]
Is this feature actively developed? How much work still need to be done to make it work on iOS?

**Comment by xuhcc** [comment]
@johan456789 I think the main problem is that the side drawer plugin doesn't work on iOS: https://gitlab.com/burke-software/nativescript-foss-sidedrawer/-/issues/7
